### PR TITLE
Update project to support Visual Studio 2017 RC

### LINKED ISSUE #101: Add support for Visual Studio 2017 RC

### PR DESCRIPTION
**Pull request by sharwell**
Fixes #101

### CODE REVIEW

File: GitDiffMargin/source.extension.vsixmanifest [hunk 1]
```diff
@@ -13,10 +13,11 @@
     <!--<Tags></Tags>-->
   </Metadata>
   <Installation>
-    <InstallationTarget Version="[11.0,15.0)" Id="Microsoft.VisualStudio.Pro" />
+    <InstallationTarget Version="[11.0,16.0)" Id="Microsoft.VisualStudio.Community" />
```
Comment (laurentkempe) [inline GitDiffMargin/source.extension.vsixmanifest]: Don't we need to have Pro and Enterprise on top of Community?

**Comment by laurentkempe** [comment]
The [following blog post](http://www.visualstudioextensibility.com/) suggest this 

> You need an updated pre-release VS SDK (via NuGet). This VS SDK generates two additional files (catalog.json and manifest.json) inside the output .vsix file (which being a .zip file really, you can rename and unzip to inspect) that were not present in the .vsix generated by the VS 2015 SDK.

Do you think it is correct

**Comment by laurentkempe** [comment]
At least now when I install it in Visual Studio 2017 RC I don't get anymore the installer message "“The following extensions are not compatible with Visual Studio 15. Installation may cause Visual Studio to be unstable”

File: GitDiffMargin/GitDiffMargin.csproj [hunk 2]
```diff
@@ -320,7 +317,7 @@
     </EmbeddedResource>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Condition="Exists($(VsSdkTargets))" Project="$(VsSdkTargets)" />
+  <Import Project="$(VSToolsPath)\VSSDK\Microsoft.VsSDK.targets" Condition="'$(VSToolsPath)' != ''" />
```
Comment (sharwell) [inline GitDiffMargin/GitDiffMargin.csproj]: @laurentkempe This line is the one that switches to the use of the new SDK.

File: GitDiffMargin/source.extension.vsixmanifest (hunk 1 above)
Comment (sharwell) [inline GitDiffMargin/source.extension.vsixmanifest]: Pro and Enterprise are implied. @madskristensen now uses a single installation target for his extensions as well. I made this change based on what I saw there.

**Comment by laurentkempe** [comment]
Again a great work @sharwell 👍 👍 👍

**Pull request merged**